plans/provision/container.fmf

```diff
@@ -1,8 +1,25 @@
-summary: Run tests in a container
+summary: Provision a container using podman
```

Comment:
Hmm, the original name sounded more fitting. "Provision a container using podman" does not say much about the plan, it feels more like a description of a single test. The sentence from `description`, "Verify functionality of the container provision plugin", sounds good, how about that one?

Reply:
Ack, makes sense, adjusted in 3fe95b6e.